configs/envoy_double_proxy_v2.template.yaml

```diff
@@ -91,20 +97,30 @@ static_resources:
     type: STATIC
     connect_timeout: 0.25s
     lb_policy: ROUND_ROBIN
-    hosts:
-      - socket_address:
-          protocol: TCP
-          address: 127.0.0.1
-          port_value: 8125
+    load_assignment:
+      cluster_name: statsd
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: 127.0.0.1
+                port_value: 8125
+                protocol: TCP
   - name: backhaul
     type: STRICT_DNS
     connect_timeout: 1s
     lb_policy: ROUND_ROBIN
-    hosts:
-    - socket_address:
-        protocol: TCP
-        address: front-proxy.yourcompany.net
-        port_value: 9400
+    load_assignment:
```

Comment:
So, this is a bit more than just switching to `Any`, can you update the commit message and PR title to cover all the things that are changing?

Reply:
It's true that I'm also changing the deprecated `hosts:` to `load_assignment:`. Will update title.

Reply:
Thanks for this though, so when I resurrect this: https://github.com/envoyproxy/envoy/issues/4618 I don't have to deal with the `host` to `load_assignment` updates for examples.